Entry.java

```diff
@@ -91,16 +78,25 @@ private boolean validate(String value){
     }
 
     /**
-     * Constructor which initializes a new purchase if details have been entered in the main class
-     * @param amount float representing the amount
-     * @param category Category where the user has spent the money
-     * @param date date of purchase
+     * Make the user enter the amount of the purchase. Give indications
+     * that is has to be a positive amount.
+     */
+    public void selectAmount(){
+        userIn = new Scanner(System.in);
```

Comment:
Would it be possible to create one scanner object as begining of class or does this screw things up?

Reply:
It should be fine. I don't think it will screw anything up.

Reply:
actualy, it might if its the same scanner as in main menu, so dont bother changing anything